recipes/gps-timemachine/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "gps-timemachine" %}
+{% set version = "1.1.5" %}
+{% set sha256 = "7758810c5417e2f7a80834d422d01f55f05709897bd06df107f19d2f8c896fa7" %}
+{% set build_number = 0 %}
```

Comment:
Please use as few as possible jinja.
```suggestion
```

Reply:
Ha! My browser did not update but Uwe was faster than me.

Reply:
Thanks Uwe and Filipe!